src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -488,7 +488,11 @@ export class Connector implements IConnector {
      * * uses `favicon.ico` and the final url after redirects.
      */
     private async getFavicon(element?: AsyncHTMLElement) {
-        const href = element ? element.getAttribute('href') : '/favicon.ico';
+        if (!element || !element.getAttribute('href')) {
```

Comment:
> Do we still want to list all the warnings/errors under this url?

@qzhou1607 Sorry, I don't quite understand the question. Can you be more specific, thanks!